helpers/GenerateResourcesAndImage.ps1

```diff
@@ -240,13 +240,19 @@ Function GenerateResourcesAndImage {
 
     try {
         # Login to Azure subscription
+        try {
+            $azAccount = az account show -o none
+            Write-Warning "Already logged in..."
+        }
```

Comment:
Could you please follow PowerShell styleguide as it is recommended in this repo readme? See https://github.com/actions/runner-images/blob/main/CONTRIBUTING.md for more details on correct style.

1) `}` and `catch` should be on the same line
2) content of `catch` block should have different indentation from `catch` block itself.